models/Post.js

```diff
@@ -17,12 +17,9 @@ Post.init(
       type: DataTypes.STRING,
       allowNull: false,
     },
-    post_url: {
-      type: DataTypes.STRING,
+    post_text: {
```

Comment:
If we don't want there to be a 500 char count limit we could use the TEXT datatype but this works for me.